src/main/scala/org/broadinstitute/dsde/rawls/expressions/SlickExpressionParsing.scala

```diff
@@ -117,9 +117,19 @@ trait SlickExpressionParser extends JavaTokenParsers {
   private def entityRootDot: Parser[RootFunc] =
     "this." ^^ { _ => entityRootFunc}
 
+  // a Java identifier or two Java identifiers separated by a colon
+  private val namespacedIdent: Parser[String] = {
+    ident ^^ {
+      case name => name
+    } |
+    ident ~ ":" ~ ident ^^ {
```

Comment:
What does "not work" mean here? Compile error? Runtime exception? The wrong behavior?